service/azureconfig/v1/resource/vnetpeering/error.go

```diff
@@ -0,0 +1,18 @@
+package vnetpeering
+
+import (
+	"github.com/giantswarm/microerror"
+)
+
+var (
+	invalidConfigError = microerror.New("invalid config")
+	wrongTypeError     = microerror.New("wrong type")
+)
```

Comment:
This is not how we do it usually. I would like to stick to our style. It makes code easier to scan when patterns stay the same.